add peripheral_proximity sample and Link Loss service (#8180)
nrfconnect/sdk-nrf: samples/bluetooth/peripheral_proximity/README.rst

> .. _peripheral_Proximity:

Reviewer: ```suggestion
.. _peripheral_proximity:
```

Bluetooth: Peripheral Proximity
###############################

.. contents::
   :local:
   :depth: 2

Overview
********

> The Proximity Application is an example that implements the Proximity Profile,The Proximity Profile alerts the user when connected devices are too far apart.

Reviewer 2: ```suggestion
The Proximity Application is an example that implements the Proximity Profile. The Proximity Profile alerts the user when connected devices are too far apart.
```

> The Proximity Application is an example that implements the Proximity Profile.The Proximity Profile alerts the user when connected devices are too far apart.

Reviewer 3: ```suggestion
The Proximity Application is an example that implements the Proximity Profile.
The Proximity Profile alerts the user when connected devices are too far apart.
```

The application includes the following services:

> For the Proximity Profile:

Reviewer 3: You can add a reference to services if it is possible

> The application includes the following services:
> 
> For the Proximity Profile:

Reviewer: ```suggestion
The application includes the following services used by the Proximity Profile:
```

> Link Loss Service
> Immediate Alert Service
> TX Power Service

Reviewer: ```suggestion
* Link Loss Service
* Immediate Alert Service
* TX Power Service
```

Requirements
************

The sample supports the following development kits:

.. table-from-sample-yaml::

The sample also requires a phone or tablet running a compatible application, for example `nRF Connect for Mobile`_ or `nRF Toolbox`_.

> 

Reviewer: Remove extra line

Building and running
********************
.. |sample path| replace:: :file:`samples/bluetooth/peripheral_proximity`

.. include:: /includes/build_and_run.txt

.. _peripheral_proximity_testing:

Testing
=======

> After programming the sample to your dongle or development kit, test it by performing the following steps:

Reviewer: ```suggestion
After programming the sample to your dongle or development kit, complete the following steps to test it:
```

1. Start the `nRF Connect for Mobile`_ application on your smartphone or tablet.
#. Power on the development kit.

> #. Connect to the device from the nRF Connect application.
>    The device is advertising as ``Nordic_Proximity``.

Reviewer: ```suggestion
#. Connect to the device from the nRF Connect application.

   The device is advertising as ``Nordic_Proximity``.
```

   The services of the connected device are shown.

> #. In :guilabel:` Immediate Alert Service`, Set the AlertLevel characteristic in Immediate Alert Service to 1 (enter '01' in the text box, and click the 'write' button), and observe the Mild alert(LED 1   is blinking (period 1200 msec, duty cycle: 50%)) .

Reviewer: ```suggestion
#. In :guilabel:`Immediate Alert Service`, set the AlertLevel characteristic to ``1`` (enter ``01`` in the text box, and click :guilabel:`Write`), and observe the Mild alert (**LED 1** is blinking (period 1200 msec, duty cycle: 50%)) .
```
Not sure whether the write button is capitalized or not. Check this to verify it.

> #. Change the AlertLevel value to 2 and observe the High alert(LED 1 is on).

Reviewer: ```suggestion
#. Change the AlertLevel value to ``2`` and observe the High alert (**LED 1** is lit).
```

> #. Change the AlertLevel value to 0 and observe LED 1 is off.

Reviewer: ```suggestion
#. Change the AlertLevel value to ``0`` and observe **LED 1** is off.
```

> #. Set the AlertLevel characteristic in Link Loss Service to 1.

Reviewer: ```suggestion
#. Set the AlertLevel characteristic in Link Loss Service to ``1``.
```

> #. Simulate a link loss situation, for example by shielding the board, and observe that LED 2 is is blinking (period 1200 msec, duty cycle: 50%).

Reviewer: ```suggestion
#. Simulate a link loss situation, for example by shielding the board, and observe that **LED 2** is is blinking (period 1200 msec, duty cycle: 50%).
```

> #. Reconnect and observe that the LED 2 is off.

Reviewer: ```suggestion
#. Reconnect and observe that the **LED 2** is off.
```

> #. Set the AlertLevel characteristic in Link Loss Service to 2.

Reviewer: ```suggestion
#. Set the AlertLevel characteristic in Link Loss Service to ``2``.
```

> #. Simulate a link loss situation, for example by shielding the board, and observe that LED 2 is on.

Reviewer: ```suggestion
#. Simulate a link loss situation, for example by shielding the board, and observe that **LED 2** is lit.
```

> #. Reconnect and observe that the LED 2 is off.

Reviewer: ```suggestion
#. Reconnect and observe that the **LED 2** is off.
```

> #. Select the TxPowerLevel characteristic in TX Power Service, click the 'read' button, and verify that it was successfully read.

Reviewer: ```suggestion
#. Select the TxPowerLevel characteristic in TX Power Service, click :guilabel:`Read`, and verify that it was successfully read.
```
Not sure whether the read button is capitalized. Please verify.

> 

Reviewer 3: ```suggestion
```

Dependencies
************

> This sample uses the following |NCS| libraries:

Reviewer 4: ```suggestion
This sample uses the following |NCS| library:
```
